napi.h

```diff
@@ -523,6 +523,8 @@ namespace Napi {
       double numberValue ///< Property value
     );
 
+    Napi::Array GetPropertyNames(); ///< Get all property names
```

Comment:
I think there's no need for the `Napi::` prefix here, and also in the implementation. Of course it doesn't hurt, but I'd prefer to keep the code less verbose when possible.